std/meta.d

```diff
@@ -1620,3 +1621,36 @@ private template Pack(T...)
 // TODO: Consider publicly exposing this, maybe even if only for better
 // understandability of error messages.
 alias Instantiate(alias Template, Params...) = Template!Params;
+
+
+/**
+Selects a subset of the argument list by stepping with fixed stepSize over the list.
+Params:
+    stepSize = Number of elements to increment on each iteration. Needs to be greater than `0`.
+    offset = Initial offset
+    Args = Template arguments
+Returns: A template argument list filtered by the selected stride.
+*/
+template Stride(size_t stepSize, size_t offset, Args...)
```

Comment:
Is there a good reason to use `size_t` instead of `int` for `stepSize` and `offset`? Currently, `Stride!(-1, 0, short, int, long)` compiles and violates the principle of least astonishment.